### ISSUE TITLE
Need help reading error given when installing Excel Plug in

### ISSUE THREAD

**Issue by Hadabadday** [dbeaver/dbeaver#34217]
### Description

I just installed Dbeaver on a new laptop.    My connections and scripts are all set and working.   I have Eclipse installed.   When I try to install the Excel Plug in I'm getting "Cannot complete the install because some dependencies are not satisfiable"     Any thoughts what I am missing?

I'm putting this in the Work With and clicking on the Office Support tab
DBeaver - Office support - https://dbeaver.io/update/office/latest/

Message given:
Cannot complete the install because some dependencies are not satisfiable
  Software being installed: a.jre.javase 18.0.0
  Software being installed: DBeaver Office Support 1.1.168.202405191503 (org.jkiss.dbeaver.ext.office.feature.feature.group 1.1.168.202405191503)
  Cannot satisfy dependency:
    From: DBeaver Office Support 1.1.169.202405191503 (org.jkiss.dbeaver.data.office 1.1.169.202405191503)
    To: osgi.ee; (&(osgi.ee=JavaSE)(version=17))
  Cannot satisfy dependency:
    From: DBeaver Office Support 1.1.168.202405191503 (org.jkiss.dbeaver.ext.office.feature.feature.group 1.1.168.202405191503)
    To: org.eclipse.equinox.p2.iu; org.jkiss.dbeaver.data.office [1.1.169.202405191503,1.1.169.202405191503]

### DBeaver Version

Version 22.2.2.202210092258

### Operating System

Windows 11

**Comment by E1izabeth** [comment]
Please try latest DBeaver version

**Comment by Hadabadday** [comment]
I had tried upgrading DBeaver prior to opening this ticket.    When upgrading, I did not elect to create a new desktop icon and thought the old icon would now point to the upgraded DBeaver.   I upgraded again but did create a new (second) desktop icon.  I was now able to apply the office update to the recent version of DBeaver.   I didnt account for the fact I can have 2 active versions of DBeaver loaded.
